pkgs/servers/sql/postgresql/ext/pg_safeupdate.nix

```
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, postgresql }:
+
+stdenv.mkDerivation rec {
+  pname = "pg-safeupdate";
+  version = "1.2";
+
+  buildInputs = [ postgresql ];
+
+  src = fetchFromGitHub {
+    owner  = "eradman";
+    repo   = pname;
+    rev    = "${pname}-${version}";
```

looks like upstream was retagged, can you update it with the new hash?
```suggestion
    rev    = version;
```
and before merge it, would you mind squashing the commits?

---

@marsam Done!